Description: Fix FTBFS with curl 7.21.7
 Remove deprecated curl/types.h header file to fix FTBFS with curl 7.21.7.
Origin: upstream, http://git.omp.am/?p=conky.git;a=patch;h=1321a41015695a75c1a73476d43c882dc270df64
Bug-Debian: http://bugs.debian.org/636367

--- a/src/ccurl_thread.c
+++ b/src/ccurl_thread.c
@@ -33,7 +33,6 @@
 #endif /* DEBUG */
 
 #include <curl/curl.h>
-#include <curl/types.h>
 #include <curl/easy.h>
 
 /*
--- a/src/eve.c
+++ b/src/eve.c
@@ -37,7 +37,6 @@
 #include <libxml/xmlwriter.h>
 
 #include <curl/curl.h>
-#include <curl/types.h>
 #include <curl/easy.h>
 
 #include <time.h>
